samples/boards/96b_argonkey/sensors/README.rst

```diff
@@ -12,9 +12,15 @@ It makes use of both the trigger and poll methods.
 Requirements
 ************
 
-This sample just requires the ArgonKey board.
+This sample just requires the ArgonKey board. The board can be fed
```

Comment:
You mention later you'll need a USB to TTL 1V8 serial cable.

Also, by "fed" do you mean "powered"?